config/internal/apiserver/default/deployment.yaml.tmpl

```diff
@@ -179,32 +179,32 @@ spec:
             {{ if .APIServer.EnableSamplePipeline }}
             - --sampleconfig=/config/sample_config.json
             {{ end }}
+            {{ if .InterPodTLS }}
+            - --tlsCertPath=/etc/tls/private/tls.crt
+            - --tlsCertKeyPath=/etc/tls/private/tls.key
```

Comment:
refresh my memory -- does OpenShift make these available here for you by default?

Reply:
no, these are generated by openshift `service-ca` operator via [this annotation](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/internal/apiserver/default/service.yaml.tmpl#L7)

[docs](https://docs.openshift.com/container-platform/4.8/security/certificates/service-serving-certificate.html)